### PR TITLE
Fix the application-type property in the oidc web authentication guide

### PR DESCRIPTION
**Pull request by sberyozkin**

### CODE REVIEW

File: docs/src/main/asciidoc/security-openid-connect-web-authentication.adoc [hunk 1]
```diff
@@ -65,11 +65,11 @@ The OpenID Connect extension allows you to define the configuration using the `a
 ----
 quarkus.oidc.auth-server-url=http://localhost:8180/auth/realms/quarkus
 quarkus.oidc.client-id=frontend
-quarkus.oidc.client-type=web-app
+quarkus.oidc.application-type=web-app
```
Comment (cescoffier) [inline docs/src/main/asciidoc/security-openid-connect-web-authentication.adoc]: Was it a typo in the doc or is it related to some recent code changes?

**Comment by cescoffier** [comment]
I've added a comment, just to be sure it's not a breaking change that needs to be tracked.

File: docs/src/main/asciidoc/security-openid-connect-web-authentication.adoc (hunk 1 above)
Comment (sberyozkin) [reply docs/src/main/asciidoc/security-openid-connect-web-authentication.adoc]: It was something we discussed with Pedro, `client-type` was in the PR only but `application-type` was what made it to the master

**Comment by sberyozkin** [comment]
@cescoffier sure, it probably made it to the docs while we were still discussing PR, so it is OK

**Review by pedroigor — APPROVED** [review]

**Review by rsvoboda — APPROVED** [review]

**Comment by geoand** [comment]
So are we good to merge this?

**Pull request merged**